app/schema.py

```diff
@@ -96,6 +96,8 @@ class Config:
 
 
 class Meeting(BaseModel):
+    """NOTE: 'Meetings' table from postgres nearly identical.
```

Comment:
The new model replaces this, lets get rid of old models as we write replacements, unless there is a valid reason to keep the old one. In this case I think deleting the old one now makes sense.

Reply:
Removed `Meeting` as suggested here. Thank you Robert.